src/VisualStudio/Core/Def/Implementation/ChangeSignature/AddParameterDialogViewModel.cs

```diff
@@ -36,28 +36,29 @@ public AddParameterDialogViewModel(Document document, int insertPosition)
 
         public string CallSiteValue { get; set; }
 
-        private string TypeNameWithoutErrorIndicator
-        {
-            get
-            {
-                return TypeSymbol!.ToDisplayString();
-            }
-        }
-
         private SymbolDisplayFormat _symbolDisplayFormat = new SymbolDisplayFormat(
             genericsOptions: SymbolDisplayGenericsOptions.IncludeTypeParameters,
             miscellaneousOptions: SymbolDisplayMiscellaneousOptions.UseSpecialTypes);
 
         public ITypeSymbol? TypeSymbol { get; set; }
 
-        public string TypeName
+        public string? TypeName
         {
             get
             {
-                return (TypeSymbol!.IsErrorType() ? "(x) " : "") + TypeSymbol!.ToDisplayString(_symbolDisplayFormat);
+                return TypeSymbol?.ToDisplayString(_symbolDisplayFormat);
             }
         }
 
+        public bool TypeBinds => !TypeSymbol!.IsErrorType();
```

Comment:
```suggestion
        public bool TypeBinds => !TypeSymbol.IsErrorType();
```

IsErrorType is an extension method.